src/bin/pg_autoctl/fsm_transition.c

```diff
@@ -1033,6 +1033,15 @@ fsm_rewind_or_init(Keeper *keeper)
 bool
 fsm_maintain_replication_slots(Keeper *keeper)
```

Comment:
maybe rename this function, since it now does more.

Reply:
New name is `fsm_prepare_for_secondary`.